library/kotlin/src/io/envoyproxy/envoymobile/RequestMapper.kt

```diff
@@ -8,7 +8,7 @@ internal fun Request.outboundHeaders(): Map<String, List<String>> {
     !entry.key.startsWith(":") && !entry.key.startsWith("x-envoy-mobile")
   })
   result.putAll(retryPolicyHeaders)
-  result[":method"] = listOf(method.stringValue())
+  result[":method"] = listOf(method.stringValue)
```

Comment:
🤔 how'd this work before? Do they work the same way @buildbreaker?

Reply:
Was a method before: https://github.com/lyft/envoy-mobile/pull/840/files#diff-5075eb53390be0c0b1fc060ee9f6f359L23-L33

Reply:
I homogenized all of the enums to use the same pattern

Reply:
Nice